### DISCUSSION TITLE
Submit a modal form without reloading the entire screen

### DISCUSSION THREAD

**Discussion by d4m** [orchidsoftware/platform#2357] [Q&A]
Hi,
I have a form for adding a new device, in this form i have a Relation component as device type, and i want to create a new device type on the fly, without leaveing the main form. But when i submit the modal form then data entered in the main form are reset because entrie screen is reloaded.

Is there any option to do that?

**Comment by Edinburgher** [comment]
@d4m Have you found a solution for this?

**Reply by d4m** [reply]
Unfortunately not.
